examples/braket_features/Verbatim_Compilation.ipynb

```diff
@@ -792,6 +792,132 @@
     "print(result.measurement_counts)"
```

Comment:
<p>In <code>device.properties</code>, you can see <code>fullyConnected=True</code>, showing that qubits are fully connected. This manifests as the device having a complete topology graph.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>